homeTask_7/homeTask_7/Program.cs

```diff
@@ -50,6 +54,46 @@ static void Main(string[] args)
                     Console.Write($"{item.GetEmployee()}");
                     ((IReview)item).CodeReview();
                 }
+                
+            }
+            //8 task
+            Console.WriteLine("-----Sort AssigneTask by Name------");
+            office.Employees.Sort(new AssigneTaskCompare());
+            foreach (Employee item_1 in office.Employees)
+            {
+                Console.Write($"{item_1.GetEmployee()}");
+                if (item_1 is IAssigner)
+                {
+                    Console.Write(" assigner\n");
+                }
+                else Console.Write("\n");
```

Comment:
it's not necessary to use else in this case
you can do `Console.Write` without else

Reply:
> it's not necessary to use else in this case you can do `Console.Write` without else

Done! I little bit changed this part.